docs/guides/external-packages.mdx

```diff
@@ -0,0 +1,102 @@
+---
+title: How to use external packages
+description: "A guide on configuring your truss to use external packages"
+---
+
+You might encounter a situation where you have to incorporate a third-party package into your truss. Truss has a few different mechanisms to support this.
+
+## Using the packages directory
+
+Each truss, when initialized, comes with a `packages` directory. Inside this directory, you can place any additional python packages that your would like to reference inside your truss.
+For example, your packages directory might look like this:
+
+```
+packages/
+    package_1/
+        subpackage/
+            script.py
+    package_2/
+        utils.py
+        another_script.py
+```
+
+You can import these packages inside your `model.py` like so:
+
+```python model.py
+from package_1.subpackage.script import run_script
+from package_2.utils import RandomClass
+
+class Model:
+    def __init__(self, **kwargs):
+        random_class = RandomClass()
+
+    def load(self):
+        run_script()
+
+    ...
+    ...
+    ...
+```
+
+These packages get bundled with your truss at build time. Because of this, it's ideal to use this method when your packages are small.
+But what if you want to reference packages *outside* your truss?
```

Comment:
Could add a bit of clarification that the external packages directory is useful for sharing a package between multiple trusses